### PR TITLE
WIP: BXMSDOC-6775 Add productized reference implementation to BO Spring Boot doc

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+. Extract the `{PRODUCT_FILE}-reference-implementation/{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaplanner-spring-boot-school-timetabling.zip` file.
+. Navigate to the `optaplanner-spring-boot-school-{MAVEN_ARTIFACT_VERSION}` directory.
+. Enter the following command to build the Spring Boot school timetabling project:
++
+[source]
+----
+mvn clean install -DskipTests
+----
++
+NOTE: You can also build with Gradle from the `src` directory.
```
Comment (dupliaka) [inline doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc]: Lets add here 
`gradle build -x test` 
instruction

**Review by dupliaka — CHANGES_REQUESTED** [review]
LGTM need a small mention on gradle build

File: doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc (hunk 1 above)
Comment (dupliaka) [reply doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc]: Sorry for messing. For 7.x we decided to exclude that functionality of building with Gradle for 7.x, Please remove gradle parts.

**Review by dupliaka — CHANGES_REQUESTED** [review]

**Review by dupliaka — APPROVED** [review]

File: _artifacts/document-attributes.adoc [hunk 2]
```diff
@@ -154,7 +155,7 @@ endif::OP[]
 :GETTING_STARTED_CASES: Getting started with case management
 :GETTING_STARTED_PROCESSES: Getting started with business processes
 :GETTING_STARTED_PLANNER: Getting started with {PLANNER}
-:CREATING_SPRING_BOOT_APPLICATIONS: Creating {PRODUCT} business applications with Spring Boot
+:CREATING_SPRING_BOOT_APPLICATIONS: Using {PLANNER} with Spring Boot to create a school timetable
```
Comment (mramendi) [inline _artifacts/document-attributes.adoc]: I think this one is wrong. CREATING_SPRING_BOOT_APPLICATIONS should refer to a different assembly, namely assembly-springboot-business-apps.adoc . The assembly that you edit, and the one about Planner, is assembly-business-optimizer-springboot.adoc .

File: _artifacts/document-attributes.adoc [hunk 3]
```diff
@@ -1,8 +1,9 @@
 
-:REBUILT: Thursday, November 05, 2020
+:REBUILT: Tuesday, December 22, 2020
 
-:ENTERPRISE_VERSION: 7.10
-:ENTERPRISE_VERSION_LONG: 7.10.0
+
+:ENTERPRISE_VERSION: 7.9
+:ENTERPRISE_VERSION_LONG: 7.9.1
```
Comment (mramendi) [inline _artifacts/document-attributes.adoc:6]: Why would you do this for master? Or is this strictly temporary, to remove confusion in SME reviews?

File: doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+. Extract the `{PRODUCT_FILE}-reference-implementation/{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaplanner-spring-boot-school-timetabling.zip` file.
+. Navigate to the `optaplanner-spring-boot-school-{MAVEN_ARTIFACT_VERSION}` directory.
+. Enter the following command to build the Spring Boot school timetabling project:
++
+[source]
+----
+mvn clean install -DskipTests
+----
+
+. Enter the following command to run the Spring Boot school timetabling project
```
Comment (mramendi) [inline doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc]: If the project uses a web UI, you also need to point a browser somewhere to access it? Alternatively if there is no UI and one needs to use curl as descreibed in chapter 27, perhaps you need to point the user to chapter 27?

File: _artifacts/document-attributes.adoc (hunk 3 above)
Comment (emmurphy1) [reply _artifacts/document-attributes.adoc:6]: Yes, I will manually fix this when I merge with the repo. I commented on this in the ba chat today.

File: doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc (hunk 4 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc]: @adupliak can you clarify this step? Do we need to give the user a URL or point to the instructions to use curl?

**Review by emmurphy1 — COMMENTED** [review]
@mramendi  good catch with the title. I've fixed it now. I need to wait for Anna to advise about the 'run' command.

File: doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc (hunk 4 above)
Comment (dupliaka) [reply doc-content/enterprise-only/optimizer/spring-boot-ref-download-proc.adoc]: That is true. The UI is available by http://localhost:8080/ Could you please add this?

**Comment by dupliaka** [comment]
> @mramendi good catch with the title. I've fixed it now. I need to wait for Anna to advise about the 'run' command.

http://localhost:8080/ would be enough.

**Review by dupliaka — APPROVED** [review]
LGTM

**Review by mramendi — APPROVED** [review]

**Pull request merged**